### PR TITLE
LRDOCS-6409 clarify that you must still configure dl.store.impl

### PR DESCRIPTION
**Pull request by jhinkey**
Hey @achaparro !

Here's more clarification on using `dl.store.impl` and the text I'm going to use in the 7.x articles.

Let me know how this is for you.

Thanks!

### CODE REVIEW

**Comment by achaparro** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: discover/deployment/articles/06-upgrading-to-liferay-7-1/02-preparing-for-upgrade-to-liferay-7-1/00-preparing-for-upgrade-to-liferay-7-1-intro.markdown [hunk 1]
```diff
@@ -112,37 +112,50 @@ provide property details and examples.
 
 ## Step 6: Configuring Your Documents and Media File Store [](id=configuring-your-documents-and-media-file-store)
 
-Your next task is to review your Documents and Media configuration. Look at
-[Document Library documentation](/discover/deployment/-/knowledge_base/7-1/document-repository-configuration)
-to see all the options in @product-ver@ before executing an upgrade process.
-There are, however, two important pieces of configuration that have changed: 
+It's time to set migrate and update your document store configuration to
```
Comment (jhinkey) [inline discover/deployment/articles/06-upgrading-to-liferay-7-1/02-preparing-for-upgrade-to-liferay-7-1/00-preparing-for-upgrade-to-liferay-7-1-intro.markdown]: Should read, "It's time to migrate and update …"

**Comment by achaparro** [comment]
Hi @jhinkey,

The documentation for the file discover/deployment/articles/06-upgrading-to-liferay-7-1/02-preparing-for-upgrade-to-liferay-7-1/00-preparing-for-upgrade-to-liferay-7-1-intro.markdown 

is pretty clear now but we just talk about the new portal-ext.properties there which does not affect to the upgrade.

My main concern is that customers forget to add "dl.store.impl" to the portal-upgrade-ext.properties (it happened several times)

Do you have any idea to try to avoid that this happens again?

Apart from this, I think we should try to rethink the order of steps for the upgrade documentation. We usually explain how to configure the new server before running the upgrade but customers do it in contrary in general. They configure the property files for the upgrade first and once it finished properly, they configure the server pointing out to the same database.

Please, let me know your thoughts about this.

**Comment by achaparro** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by achaparro** [comment]
Pull request submitted to https://github.com/jhinkey/liferay-docs/pull/445. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by jhinkey** [comment]
Please see my comments in https://github.com/achaparro/liferay-docs/pull/33

**Pull request closed without merging**